source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,181 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v6.0.
 
 Latest Mattermost Releases:
-- [Release v7.6 - Feature Release](#release-v7-6-feature-release)
+
+- [Release v7.7 - Feature Release](#release-v7-7-feature-release)
 - [Release v7.5 - Feature Release](#release-v7-5-feature-release)
 - [Release v7.4 - Feature Release](#release-v7-4-feature-release)
-- [Release v7.3 - Feature Release](#release-v7-3-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
 
+## Release v7.7 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release Day: January 16, 2023**
+
+### Important Upgrade Notes
+
+ - Plugins with a webapp component may need to be updated to work with Mattermost v7.7 release and the updated ``React v17`` dependency. 
+ 	- This is to avoid plugins crashing with an error about ``findDOMNode`` being called on an unmounted component. While our [starter template](https://github.com/mattermost/mattermost-plugin-starter-template) depended on an external version of ``React``, it did not do the same for ``ReactDOM``. Plugins need to update their ``webpack.config.js`` directives to externalize ``ReactDOM``. For reference, see https://github.com/mattermost/mattermost-plugin-playbooks/pull/1489. Server-side only plugins are unaffected. This change can be done for existing plugins any time prior to upgrading to Mattermost v7.7 and is backwards compatible with older versions of Mattermost. If you run into issues, you can either enable ``ExperimentalSettings.PatchPluginsReactDOM`` or just disable the affected plugin while it's updated.
+ - Denormalized ``Threads table`` by adding the ``TeamId`` column. See details for schema changes in the [Important Upgrade Notes](/upgrade/important-upgrade-notes.html).
+
+**IMPORTANT:** If you upgrade from a release earlier than v7.5, please read the other [Important Upgrade Notes](/upgrade/important-upgrade-notes.html).
+
+### Compatibility
+ - Updated the minimum version of MacOS to 11+.
+
+### Highlights
```

Comment:
@johndavidlugtu should in-product purchase be added as a feature highlight for v7.7?

Reply:
Hmm.. are we certain that we can ship it by 7.7 @marianunez based on potential feedback from security?

Reply:
If we decide to pull the feature we can remove from the release notes. Just wanted to make sure we didn't miss the highlight if we do ship it

Reply:
Good point -- heads up @amyblais that we _might_ need to pull this from 7.7

Reply:
@amyblais heads up the status on this feature In-product Purchase is that it is included in v7.7 but it's availability is controlled by a feature flag in the portal. The feature will be disabled until we get a clear from security. cc/ @DSchalla